### ISSUE TITLE
Disable the JS Scrolling on the consolidated dashboard + analyze page

### ISSUE THREAD

**Issue by SeanKennyNF** [serotracker/dashboard#178]
Let's disable the JS Scrolling on the consolidated dashboard + analyze page and rely solely on the CSS scrolling. The css scrolling comes from these css classes on the divs `snap-start snap-always scroll-smooth` so as long as those are still there you're good.

- The big component that enables the JS scrolling is called `ScrollSectionGroup` and lives in `components/ui/scroll-section-group/scroll-section-group.tsx`. This is the component that has to go.
- Instead of using url search params to determine which section to scroll to, let's use the hash property to link to sections with certain ids in the html ([this is new to me but this looks like the right property](https://developer.mozilla.org/en-US/docs/Web/API/URL/hash))
- Basically if you just take every section in `redesignedArbovirusPageSections` and move them to where the `renderScrollSectionGroup` call is, I feel like all you'd have to do after that is clean up unused code and add ids to the sections in the html.
